allure-app-under-test/src/test/java/com/capgemini/ntc/selenium/projectY/FormAuthenticationTest.java

```diff
@@ -0,0 +1,122 @@
+package com.capgemini.ntc.selenium.projectY;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.UUID;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import com.capgemini.ntc.core.groupTestCases.testSuites.tags.TestsSelenium;
+import com.capgemini.ntc.selenium.pages.projectY.FormAuthenticationPage;
+import com.capgemini.ntc.selenium.pages.projectY.TheInternetPage;
+import com.capgemini.ntc.test.core.BaseTest;
+import com.capgemini.ntc.test.core.logger.BFLogger;
+
+@Category({ TestsSelenium.class })
+public class FormAuthenticationTest extends BaseTest {
+	
+	private static TheInternetPage			theInternetPage;
+	private static FormAuthenticationPage	formAuthenticationPage;
+	private String							errorUsernameMessage	= "Your username is invalid!\n" + "×";
+	private String							errorPasswordMessage	= "Your password is invalid!\n" + "×";
+	private String							loginMessage			= "You logged into a secure area!\n" + "×";
+	private String							logoutMessage			= "You logged out of the secure area!\n" + "×";
+	private String							emptyUsername			= "";
+	private String							emptyUserPassword		= "";
+	private String							validUsername			= "tomsmith";
+	private String							validPassword			= "SuperSecretPassword!";
+	private String							randomUsername			= UUID.randomUUID()
+					.toString();
+	private String							randomUserPassword		= UUID.randomUUID()
+					.toString();;
+	
+	@BeforeClass
+	public static void setUpBeforeClass() throws Exception {
+		BFLogger.logDebug("Step 1: Open the Url http://the-internet.herokuapp.com/ with maximized window");
+		theInternetPage = new TheInternetPage();
+		theInternetPage.getDriver()
+						.manage()
+						.window()
+						.maximize();
```

Comment:
maximize()  probably does not work with SeleniumGrid.  Please talk with @tompalka